applications/asset_tracker/prj_nrf9160_pca20035ns.conf

```diff
@@ -95,3 +99,17 @@ CONFIG_HEAP_MEM_POOL_SIZE=16384
 CONFIG_MAIN_STACK_SIZE=8192
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
 CONFIG_HW_STACK_PROTECTION=y
+
+# Settings
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+
+# MCUBOOT
+CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_IMG_MANAGER=y
+CONFIG_MCUBOOT_IMG_MANAGER=yy
```

Comment:
Files should end with a newline. Also, s/yy/y/.

Reply:
fixed